.gitignore

```diff
@@ -93,10 +93,14 @@ npm-debug.log
 /pegasus/config/newrelic.yml
 /pegasus/.sass-cache
 /pegasus/sites.v3/*/public/css/generated
+/pybin/
```

Comment:
can this be removed now? I don't see anything here in my local filesystem after running through the setup steps and pycall_test.rb.